examples/jax/debugging/simple_spread/feedforward/decentralised/run_ippo_restore_checkpoint.py

```diff
@@ -0,0 +1,119 @@
+# python3
+# Copyright 2021 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Example running IPPO on debug MPE environments."""
+
+import functools
+from typing import Any
+
+import optax
+from absl import app, flags
+
+from mava.systems.jax import ippo
+from mava.utils.environments import debugging_utils
+from mava.utils.loggers import logger_utils
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string(
+    "env_name",
+    "simple_spread",
+    "Debugging environment name (str).",
+)
+flags.DEFINE_string(
+    "action_space",
+    "discrete",
+    "Environment action space type (str).",
+)
+
+flags.DEFINE_string("base_dir", "~/mava", "Base dir to store experiments.")
+
+flags.DEFINE_string(
+    "mava_id",
+    "exp_id",
+    "Experiment identifier that can be used to continue experiments.",
+)
+
+
+def main(_: Any) -> None:
+    """Run main script
```

Comment:
This looks like the normal `run_ippo.py` example except for the comment on how to load checkpoints. Is this intended?

Reply:
I see, so when this is run first it will create a checkpoint locally and then training can be continued from that checkpoint later on? Can we maybe just add a comment to the file explaining this?

Reply:
Okay, I will do that. What actually happens is that it checks in the experiment_path if a checkpoint folder exists: if it does, it will restore the parameters from the file, if it does not (like other example files), it will not restore and continue. For all files, new checkpoints will be saved in that folder

Reply:
I also have this explanation in the documentation PR: https://github.com/instadeepai/Mava/pull/755

Reply:
Added a comment to the file @RuanJohn

Reply:
Thank you @AsadJeewa :)